src/Controllers/SignUpController.php

```diff
@@ -35,20 +35,19 @@ public function __invoke(Request $request, Response $response, ?array $args = []
     
 
          $userverify = $this->em->getRepository(User::class)->findOneBy([
-    'username' => $username
+    'username' => $username,
+    'email'    => $email
 ]);
-        if (!is_null($userverify->username())||($userverify->email()))
+        if (!is_null($userverify->username) and ($userverify->email))
```

Comment:
aquí pasa algo con la comprobación si quito () me dice  'Call to a member function email() on null' le añado () pasa la verificación vuelvo a enviar el post y sale 'Cannot access private property App\Domain\User::$email',por otra parte solo verifica el usuario si hago un post con usuario inexistente pero un email existente salta la verificaccion